README.md

```diff
@@ -53,30 +53,46 @@ repository:
 conda install -c http://use.yt/with_conda yt
 ```
 
-To get set up with a development version, you can clone this repository and
-install like this:
+To get set up with a development version, you want to clone this repository:
 
 ```
 git clone https://github.com/yt-project/yt yt-git
 cd yt-git
-pip install -e .
 ```
 
-To set up yt in a virtualenv (and there are [many good
-reasons](https://packaging.python.org/installing/#creating-virtual-environments)
-to do so!) you can follow this prescription:
+and work within a conda environment:
+
+```
+# Create a conda environment named yt-git
+$ conda create -n yt-git python=3.6
+# Activate it
+$ source activate yt-git
+# Make sure you run the latest version of conda
+$ conda update conda
+# Install only yt's dependencies
+$ conda install -c conda-forge yt --only-deps
+$ conda install -c conda-forge cython
+# Make sure you run the latest version of pip
+$ pip install --upgrade pip
+$ pip install -e .
```

Comment:
There's some important logging information that by default pip suppresses. Let's suggest `pip install -v -e .`, which will allow the output to more closely match the output of `python setup.py develop` and also avoid possible confusion from waiting for yt to compile without any new output from `pip`.